rcsb/utils/repository/RepositoryProvider.py

```diff
@@ -243,9 +243,8 @@ def __getLocatorObjListWithInput(self, contentType, inputPathList=None, mergeCon
                         idCode = fn[:4] if fn and len(fn) >= 8 else None
                         mergeLocator = self.__getLocator(mergeContentType, idCode, checkExists=True) if idCode else None
                         if mergeLocator:
-                            # kwD = HashableDict({"marshalHelper": vrd.toCif})
-                            kwD = HashableDict({"marshalHelper": toCifWrapper})
-                            oL.append(HashableDict({"locator": mergeLocator, "fmt": "xml", "kwargs": kwD}))
+                            kwD = HashableDict({})
```

Comment:
If this is already defined earlier in this loop (see line 239), is there really a need to redefine it?